dds_web/__init__.py

```diff
@@ -59,6 +61,13 @@ def create_app():
     db.init_app(app)  # Initialize database
     # ma.init_app(app)
 
+    # initialize OIDC
+    oauth.register("default_login",
```

Comment:
Can you run `black -l 100` quickly please? CI isn't added to this repo yet, but will be soon.. (same setup as `dds_cli`)